lib/lightning_web/live/run_live/index.html.heex

```diff
@@ -34,10 +34,9 @@
                     >
                       <span>
                         <%= if workflow_id = get_change(@filters_changeset, :workflow_id) do %>
-                          <%= Enum.find(@workflows, fn {_name, id} ->
-                            id == workflow_id
-                          end)
-                          |> elem(0) %>
+                          <%= Enum.find_value(@workflows, fn {name, id} ->
+                            if id == workflow_id, do: name
+                          end) %>
```

Comment:
We should change this to be a single function or assign, something that LiveView can compare arguments by value. This will get recomputed as different every render.

I know you didn't write it, but I reckon we shouldn't walk past it this time around.